inc/contact-section/contact-section.php

```diff
@@ -192,6 +192,7 @@ public static function get_option() {
 			'tel_number'        => '000-000-0000',
 			'contact_time'      => __( 'Office hours 9:00 - 18:00 [ Weekdays except holidays ]', 'vk-all-in-one-expansion-unit' ),
 			'contact_link'      => home_url(),
+			'contact_target'    => false,
```

Comment:
```suggestion
			'contact_target_blank'    => false,
```
保存値が bool なので contact_target だけの場合は true が self という誤解釈の余地があるため。
保存値が self / blank だった場合なら contact_target のままで可